reactbot.py

```diff
@@ -0,0 +1,340 @@
+# -*- coding: utf-8 -*-
+"""
+This module runs basically everything.
+
+Attributes:
+    VERSION = "2.0.0" (String): Version Number: release.version_num.revision_num
+
+    # Config Variables
+    EMOJI_LIST (List): List of Strings for emojis to be added to announcements
+    USER_LIST (JSON): List of users in JSON format
+    ADMIN (List): ["U25PPE8HH", "U262D4BT6", "U0LAMSXUM", "U3EAHHF40"] testing defaults
+    TTPB (String): Config variable, sets channel for cleverbot integration
+    GENERAL (Stirng): Config variable, sets channel for general's name
+    LOG (boolean): Global Variable
+    LOGC (boolean): Global Variable
+
+.. _Google Python Style Guide:
+   http://google.github.io/styleguide/pyguide.html
+
+"""
+
+from slackclient import SlackClient
```

Comment:
Yes, I had some trouble making it work without. We can remove it if it isnt needed on our instance but I want it to work on my machine :heh: